ss58/mod.test.ts

```diff
@@ -1,15 +1,64 @@
-import { assertEquals } from "../deps/std/testing/asserts.ts";
-import { Ss58 } from "./mod.ts";
+import { assertEquals, assertThrows } from "../deps/std/testing/asserts.ts";
+import * as p from "../test-util/pairs.ts";
 
-Deno.test("Decode Ss58 Text", async () => {
-  const decoded = (await Ss58()).decode("12Y8b4C9ar162cBgycxYgxxHG7cLVs8gre9Y5xeMjW3izqer");
-  assertEquals(decoded, [0, "43fa61b298e82f9f207ddea327900cee26b554756c4a533f36cd875e3e7bcf06"]);
+import {
+  decode,
+  decodeRaw,
+  encode,
+  InvalidAddressChecksumError,
+  InvalidAddressLengthError,
+  InvalidNetworkPrefixError,
+  InvalidPublicKeyLengthError,
+} from "./mod.ts";
+
+for (
+  const [networkName, address, [prefix, publicKey]] of [
+    [
+      "polkadot",
+      "15oF4uVJwmo4TdGW7VfQxNLavjCXviqxT9S1MgbjMNHr6Sp5",
+      [0, p.alice.publicKey],
+    ],
+    [
+      "substrate",
+      "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY",
+      [42, p.alice.publicKey],
+    ],
+    [
+      "aventus",
+      "cLxkfNUiCYsb57YLhTJdNVKxUTB1VTpeygYZNhYuFc83KrFy7", // cspell:disable-line
```

Comment:
Let's avoid placing cspell-disabling comments in the source –– hopefully you can do this in your workspace settings?

Reply:
I'll add it to the cspell config